src/components/composer/composer/composer.ts

```diff
@@ -595,13 +595,18 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
       if (tokenAtCursor) {
         const { xc } = splitReference(tokenAtCursor.value);
         if (
-          tokenAtCursor.type === "FUNCTION" ||
-          (tokenAtCursor.type === "SYMBOL" && !rangeReference.test(xc))
+          (tokenAtCursor.type === "FUNCTION" ||
+            (tokenAtCursor.type === "SYMBOL" && !rangeReference.test(xc))) &&
+          !this.env.model.getters.getCurrentTokens().some((t) => t.type === "UNKNOWN")
```

Comment:
auto-complete should not be disabled. Starting with 18.0, we have more auto-completes (pivots, lists)
and "état" is a valid would be a valid search if you're searching for the `state` field